lib/src/catcher.rs

```diff
@@ -239,6 +239,9 @@ pub mod defaults {
                 method, or it lacks the ability to fulfill the request.", handle_501,
             503, "Service Unavailable", "The server is currently unavailable.",
                 handle_503,
+            504, "Gateway Timeout", "The server didn't receive a timely response
```

Comment:
Let's change the following: "didn't" to "did not", and "another" to "an upstream". Let's also merge the last two lines together.

Reply:
Aye! Pushing a new commit.